mmdet/datasets/__init__.py

```diff
@@ -2,6 +2,7 @@
 from .builder import DATASETS, PIPELINES, build_dataloader, build_dataset
 from .cityscapes import CityscapesDataset
 from .coco import CocoDataset
+from.coco_5labels import CocoDataset_5labels
```

Comment:
To train only 5 labels, you do not need to implement it yourself. You only need to set classes of 5 labels in the config.